src/app/technical-record/test-history/test-history.component.ts

```diff
@@ -8,28 +9,23 @@ import {TechRecordHelpersService} from '@app/technical-record/tech-record-helper
   changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class TestHistoryComponent implements OnInit {
+  @Input() testResultJson: TestResultModel[];
 
-  @Input() testResultJson: any;
+  constructor(public techRecHelpers: TechRecordHelpersService) {}
 
-
-  constructor(public techRecHelpers: TechRecordHelpersService) {
-  }
-
-  ngOnInit() {
-  }
+  ngOnInit() {}
 
   setResultColor(testResult: string) {
```

Comment:
Can we use scss classes instead of doing it with js.
It will be easier to maintain and also in case the colours change, the snapshot will tell us and this will help for regression